corehq/apps/api/odata/utils.py

```diff
@@ -1,15 +1,19 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+from collections import defaultdict
+
 from corehq.apps.export.models import CaseExportDataSchema
 from corehq.apps.reports.analytics.esaccessors import get_case_types_for_domain_es
 
 
 def get_case_type_to_properties(domain):
-    case_type_to_properties = {}
+    case_type_to_properties = defaultdict(list)
     case_types = get_case_types_for_domain_es(domain)
     for case_type in case_types:
-        case_type_to_properties[case_type] = []
+        if not case_type:
+            # TODO - understand why a case can have a blank case type and handle appropriately
```

Comment:
Think it's appropriate to soft assert for this or anything?  I suppose it's ES, so it'd be pretty easy to query later when doing analysis.

Reply:
Probably don't need a soft assert because this function is going to be replaced.